coordinator/src/services/translation/request/dispense/release.ts

```diff
@@ -40,23 +56,39 @@ function getAgentPersonPerson(): hl7v3.AgentPersonPerson {
   return agentPerson
 }
 
-function getRepresentedOrganization(fhirReleaseRequest: fhir.Parameters): hl7v3.Organization {
-  const hl7Organization = new hl7v3.Organization()
-
+async function getRepresentedOrganization(
+  fhirReleaseRequest: fhir.Parameters,
```

Comment:
might be nice if this gets an organization from its code rather than a parameter list (so a code gets passed in)